frontend/app/routes/$lang/_public/status/child.tsx

```diff
@@ -214,6 +214,8 @@ export async function action({ context: { session }, params, request }: ActionFu
         dateOfBirth: parsedChildInfoResult?.data.dateOfBirth ?? '',
       });
 
+  statusId = statusId === INVALID_CLIENT_FRIENDLY_STATUS ? null : statusId;
+
   const clientFriendlyStatus = statusId ? lookupService.getClientFriendlyStatusById(statusId) : null;
```

Comment:
you could get away with re-binding `statusId` and keeping it `const` by declaring `const clientFriendlyStatus = statusId && statusId !== INVALID_CLIENT_FRIENDLY_STATUS ? lookupService.getClientFriendlyStatusById(statusId) : null;`

saves a line too

Reply:
We still need this line `
  statusId = statusId === INVALID_CLIENT_FRIENDLY_STATUS ? null : statusId;`

since it assigns `null` to `statusId`.

Reply:
yes, what I'm saying is

```
const statusId = parsedSinResult
    ? await applicationStatusService.getStatusIdWithSin({
        sin: parsedSinResult.data.sin,
        applicationCode: parsedCodeResult.data.code,
      })
    : await applicationStatusService.getStatusIdWithoutSin({
        applicationCode: parsedCodeResult.data.code,
        firstName: parsedChildInfoResult?.data.firstName ?? '',
        lastName: parsedChildInfoResult?.data.lastName ?? '',
        dateOfBirth: parsedChildInfoResult?.data.dateOfBirth ?? '',
      });
const clientFriendlyStatus = statusId && statusId !== INVALID_CLIENT_FRIENDLY_STATUS ? lookupService.getClientFriendlyStatusById(statusId) : null;
```

Reply:
I understand but this does not assign null to statusId.

Since statusId is used in the code here:
![image](https://github.com/user-attachments/assets/4b2741b0-e6b9-4cc9-a64b-226e40488025)

 We still need this line:
 `statusId = statusId === INVALID_CLIENT_FRIENDLY_STATUS ? null : statusId;
`

Reply:
Good point!